llvm/test/Transforms/InstCombine/div.ll

```diff
@@ -1432,6 +1432,47 @@ define <2 x i8> @sdiv_sdiv_mul_nsw(<2 x i8> %x, <2 x i8> %y, <2 x i8> %z) {
   ret <2 x i8> %r
 }
 
+; -(X * Y) / (X * Y) --> -1
+define i32 @sdiv_neg_mul_mul(i32 %0, i32 %1) {
+; CHECK-LABEL: @sdiv_neg_mul_mul(
+; CHECK-NEXT:    ret i32 -1
```

Comment:
These don't look properly generated -- shouldn't there be an extra `;` line here?